lux/test/unit/lux/prisms/goat_sdk/uniswap_swap_test.exs

```diff
@@ -0,0 +1,124 @@
+defmodule Lux.Prisms.GoatSdk.UniswapSwapTest do
+  use UnitCase, async: true
+
+  alias Lux.Prisms.GoatSdk.UniswapSwap
+
+  @moduletag :unit
+
+  setup do
+    mock_python_code = """
```

Comment:
`setup do` blocks run before each `test`. Was this the goal here? If you need this code to exist in the system we could add a python file inside `lux/privy/python/goat`